lib/BC/SleighLifter.cpp

```diff
@@ -429,16 +433,20 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock : public PcodeEmit {
       case OpCode::CPUI_CALL: {
         // directs dont read the address of the variable, the offset is the jump
         // TODO(Ian): handle other address spaces
-
+        if (input_var.getAddr().getSpace()->constant_space_index ==
```

Comment:
This feels like a really leaky abstraction, since we need to go through a lot of random calls to get some info - provide a helper function?

Reply:
I mean it's sleigh's abstraction not ours :P,  but ive added a helper function that determines if a vnode is in the constant address space